sdk/eventhub/Azure.Messaging.EventHubs/src/Metadata/PartitionProperties.cs

```diff
@@ -73,23 +73,23 @@ public sealed class PartitionProperties
         /// <param name="beginningSequenceNumber">The first sequence number available for events in the partition.</param>
         /// <param name="lastSequenceNumber">The sequence number observed the last event to be enqueued in the partition.</param>
         /// <param name="lastOffset">The offset of the last event to be enqueued in the partition.</param>
-        /// <param name="lastEnqueueUtc">The date and time, in UTC, that the last event was enqueued in the partition.</param>
+        /// <param name="lastEnqueuedTime">The date and time that the last event was enqueued in the partition.</param>
```

Comment:
Let's please keep the ", in UTC," call outs in the comments.

Reply:
Done.